avocado/utils/linux_modules.py

```diff
@@ -119,7 +119,12 @@ def loaded_module_info(module_name):
                 elif key == 'depends':
                     value = items[1].split(',')
                 elif key == 'parm':
-                    param_list.append(items[1].split(':')[0])
+                    param_dic = {'type': None}
+                    param_dic['name'] = items[1].split(':')[0]
+                    param_type = re.search(r"\((\w+)\)", items[-1])
+                    if param_type is not None:
+                        param_dic['type'] = param_type.group(1)
+                    param_list.append(param_dic)
```

Comment:
@ldoktor Agreed. Will make sure to keep that in mind in future, though I could not work out a format which is backward compatible for this one.